src/cephadm/cephadm

```diff
@@ -3842,7 +3857,10 @@ def list_networks(ctx):
 
 
 def _list_ipv4_networks(ctx: CephadmContext):
-    out, _, _ = call_throws(ctx, [find_executable('ip'), 'route', 'ls'])
+    execstr: Optional[str] = find_executable('ip')
```

Comment:
(fyi, you typically don't need to annotate variables that are returned by other functions)

Reply:
Just for clarity. I really do love being as verbose about types as possible :)